test/parallel/test-buffer-tojson.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const Buffer = require('buffer').Buffer;
```

Comment:
as a best practice yeah. I believe at one point there was a linting error reported if it wasn't explicitly required. Not sure if that's still in place tho.